app/Jobs/ApplicationDeploymentJob.php

```
@@ -2436,49 +2656,207 @@ private function generate_build_env_variables()
+
+        if ($modified) {
+            // Write the modified Dockerfile back
+            $dockerfile_base64 = base64_encode($dockerfile->implode("\n"));
+            $this->execute_remote_command([
+                executeInDocker($this->deployment_uuid, "echo '{$dockerfile_base64}' | base64 -d | tee {$dockerfile_path} > /dev/null"),
+                'hidden' => true,
+            ]);
+
+            $this->application_deployment_queue->addLogEntry('Modified Dockerfile to use build secrets.');
+        }
+    }
```

PHPMD (2.15.0)

The method modify_dockerfile_for_secrets is not named in camelCase. (undefined)

(CamelCaseMethodName)